joblib/_memmapping_reducer.py

```diff
@@ -570,17 +570,22 @@ def resolve_temp_folder_name(self):
 
```

Comment:
- [ ] changes in this file are better addressed by https://github.com/joblib/joblib/pull/1391 . When those are merged, the diff here should be reverted.